src/sentry/utils/sdk.py

```diff
@@ -21,7 +22,16 @@
 from sentry.utils import metrics
 from sentry.utils.rust import RustInfoIntegration
 
-UNSAFE_FILES = ("sentry/event_manager.py", "sentry/tasks/process_buffer.py")
+UNSAFE_FILES = (
+    "sentry/web/api.py",
+    "sentry/event_manager.py",
+    "sentry/tasks/process_buffer.py",
+    "sentry/tasks/post_process.py",
+    "sentry/tasks/store.py",
+    "sentry/ingest/ingest_consumer.py",
+    "sentry/ingest/outcomes_consumer.py",
+    "sentry/eventstream/kafka/backend.py",
```

Comment:
cc @tkaemming, @fpacifici: This will require you to check S4S.

Reply:
👍 Thanks for the heads up